### DISCUSSION TITLE
10월 31일 포스팅 글부터 목록에서  대표 이미지가 정말 작게 나옵니다.

### DISCUSSION THREAD

**Discussion by Ryan-Dia** [pronist/hELLO#142] [Q&A] [answered]
운영체제: macOS Monterey
브라우저: chrome
버전:107.0.5304.87(공식 빌드) (arm64)
스킨 버전: 3.6.3

설명
이전글을 그대로인데 10월30일부터 쓴 글부터 저렇게 됩니다. 
갑자기 'element.style'  css가 적용되서 그런 것 같습니다.
이건 티스토리에서 패치해서 그런건가요 ? 
10월 31일 전에는 element.style가 적용이 되어있지 않습니다.
이전 글은 기본에 사진이 없던 글이나 사진이 이미 있어도 교체를 해도 잘되지만 10월 30일 이후 글은 저렇게 작게나옵니다. 

발생 과정
이미지가 아주아주 작게 나옵니다. 

스크린샷 및 링크

블로그 링크 : https://html-jc.tistory.com/?page=1

<img width="1164" alt="image" src="https://user-images.githubusercontent.com/76567238/199298049-bf5cca61-5b4a-4034-8eef-9ba6f9263739.png">

<img width="665" alt="image" src="https://user-images.githubusercontent.com/76567238/199298284-bac28612-ecac-482a-8e2a-c40a68ed6643.png">

기본 상태

<img width="1034" alt="image" src="https://user-images.githubusercontent.com/76567238/199299781-702c6aa3-dd56-47e6-a6f9-01fbabad4cfd.png">

element.style 전부 적용 해제

<img width="978" alt="image" src="https://user-images.githubusercontent.com/76567238/199298459-3c218339-c99a-453b-9290-7d544605697a.png">

**Comment by F-KAZ** [comment]
저만 그런것이 아니었군요..  PC 버전에서만 그러네요! (스킨편집, 모바일은 정상)

**Comment by pronist** [comment]
@Ryan-Dia @F-KAZ 안녕하세요. 정상우입니다.

카카오 티스토리 팀에서 또 뭘 건드린 건지 제가 오늘 글쓰고 테스트 해봐도 동일한 증상이 나타나는 것을 파악하였습니다.
상황을 봐서는 인라인 스타일로 적용된 것으로 보아 
일시적인 버그일 것으로 추정됩니다만, 실제로 문제가 발생한 것은 맞기 때문에 조치를 취해야겠습니다.

감사합니다.

**Comment by pronist** [comment, marked as answer]
@Ryan-Dia @F-KAZ 

`3.6.4` 버전에서 문제를 해결하고 다시 배포하였습니다. 혹여나 해결되지 않은 경우에는 댓글 남겨주시기 바랍니다.

**Reply by Ryan-Dia** [reply]
이제 잘 잘동합니다.  대단히 감사합니다.
